src/mn_setup.c

```diff
@@ -1121,14 +1121,33 @@ static void DrawGyroCalibration(void)
 //
 // killough 8/15/98: rewritten
 
+typedef enum
+{
+    MENU_HELP_OFF,
+    MENU_HELP_AUTO,
+    MENU_HELP_KEY,
+    MENU_HELP_PAD
+} menu_help_t;
+
+static menu_help_t menu_help;
+
+static void PRINTF_ATTR(2, 3) HelpString(const char **s, const char *fmt, ...)
```

Comment:
Not 100% sure about this use of `PRINTF_ATTR`.

Reply:
It looks correct. If something is wrong with this attribute, gcc/clang will always report it.